src/conditional-flow/boolean-conditions.js

```diff
@@ -1,12 +1,16 @@
 // This function should accept a boolean value and return the string
 // "Well done, you passed!" if the value is true, or "Sorry, try again"
 // if the value is false.
-function getResult (didPass) {
+function getResult(didPass) {
+  if (didPass == true) {
+    return "Well done, you passed!";
+  } else {
```

Comment:
You don't really need the else here, you could just have the 
```js
if (didPass == true) {
  return "Well done, you passed!";
}
return "Sorry, try again";
```
This is because `didPass` will either be true or false. So in the case it is true, your function will stop when you return Well done. In the case it is false, it will skip the if statement and go onto the next line; in the code I showed above, it'll return Sorry.

Hope that makes sense!